ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -161,28 +174,20 @@ const mapStateToProps = (state, ownProps) => {
     to: txParamsToAddress,
     gasPrice,
     gas: gasLimit,
-    value: amount,
     data,
   } = (transaction && transaction.txParams) || txParams;
   const accounts = getMetaMaskAccounts(state);
   const smartTransactionsOptInStatus = getSmartTransactionsOptInStatus(state);
   const currentChainSupportsSmartTransactions =
     getCurrentChainSupportsSmartTransactions(state);
 
-  const transactionData = parseStandardTokenTransactionData(data);
-  const tokenToAddress = getTokenAddressParam(transactionData);
```

Comment:
Is this function still used elsewhere so we can't remove it?

Reply:
This function is currently utilized within `confirm-approve.util.js`, `useTransactionDisplayData.js` and `token-util.js`. I need to look deeper into those places and try to change the recipient address to the same as with the contract interactions.